runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/block/FileBlock.java

```diff
@@ -69,6 +69,7 @@
   private final String filePath;
   private final FileMetadata<K> metadata;
   private final MemoryPoolAssigner memoryPoolAssigner;
+  private final static String alreadyCommited = "The partition is already committed!";  
```

Comment:
The naming convention for a final static variable is to write them in capital letters. 
It would be better to rename it to `ALREADY_COMMITTED`